### ISSUE TITLE
Feature Request: Limit scope of CSS styles

### ISSUE THREAD

**Issue by sathomas** [coolwanglu/pdf2htmlEX#261]
Although it doesn't matter for standalone web pages, in those cases where pdf2htmlEX content is being embedded in another page, it would be nice to have an option to limit the scope of CSS styles. For example, the current rule in `base.css`:

```
span {
    display: inline-block;
}
```

affects all `<span>` elements on the parent page, even those outside of the pdf2htmlEX content. It would be helpful if it were possible (e.g. as a run-time option) to enclose all the pdf2htmlEX styles in a parent selector, e.g.:

```
.pdf2htmlex span {
    display: inline-block;
}
```

It wouldn't even have to be an option if, e.g., the `pdf2htmlex` class were simply added to the `<body>` tag.

**Comment by Helidium** [comment]
@coolwanglu Is there any progress on this issue? I have a situation, where I need to display several pdf files on the same page.

**Comment by duanyao** [comment]
@Helidium No progress yet. Is iframe feasible in your case?

**Comment by Helidium** [comment]
@duanyao  Not really, as individual documents represent parts of one whole document that must be displayed alltogether. If you can help me by stating where css file gets generated I can implement such functionality according to input parameter.

**Comment by duanyao** [comment]
@Helidium iframes can be considered as parts of their parent document. You may also consider merge those PDFs before converting to html, if they are actually parts of one document.

For css generation codes, look at [`all_manager.xxx.install()`](https://github.com/coolwanglu/pdf2htmlEX/blob/2edf41eaa1234298241e05b66b6cbadb77c0bd93/src/HTMLTextLine.cc#L159). Maybe you can add a random number to those classes for each conversion to avoid collision.
